From 1e7696809a7d86f1bec9261ee42dc52e8ed9f1db Mon Sep 17 00:00:00 2001
From: Joris Vaillant <joris.vaillant@inria.fr>
Date: Fri, 31 May 2024 14:56:59 +0200
Subject: [PATCH] cmake: CMake don't check anymore for dependency bindings
 existence

---
 CMakeLists.txt | 37 ++-----------------------------------
 1 file changed, 2 insertions(+), 35 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 99ab2ce2..878a87f2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -184,44 +184,11 @@ export_boost_default_options()
 add_project_dependency(Boost REQUIRED COMPONENTS ${BOOST_REQUIRED_COMPONENTS})
 
 if(BUILD_PYTHON_INTERFACE)
-  set(PYTHON_COMPONENTS Interpreter Development.Module NumPy Development)
+  set(PYTHON_COMPONENTS Interpreter Development NumPy)
   add_project_dependency(eigenpy 3.4.0 REQUIRED PKG_CONFIG_REQUIRES "eigenpy >= 3.4.0")
   set(PYLIB_NAME "py${PROJECT_NAME}")
   set(${PYLIB_NAME}_INSTALL_DIR ${PYTHON_SITELIB}/${PROJECT_NAME})
-
-  execute_process(
-    COMMAND ${PYTHON_EXECUTABLE} -c "import platform; print(platform.python_implementation())"
-    OUTPUT_VARIABLE _python_implementation_value
-    OUTPUT_STRIP_TRAILING_WHITESPACE ERROR_QUIET
-  )
-  message(STATUS "Python compiler: ${_python_implementation_value}")
-
-  execute_process(
-    COMMAND ${PYTHON_EXECUTABLE} -c "import proxsuite_nlp"
-    RESULT_VARIABLE _proxsuite_nlp_python_bindings_not_found
-    OUTPUT_QUIET ERROR_QUIET
-  )
-  if(_proxsuite_nlp_python_bindings_not_found EQUAL 0)
-    message(STATUS "Found proxsuite_nlp Python bindings.")
-  else()
-    message(FATAL_ERROR "proxsuite_nlp Python bindings NOT found.")
-  endif()
-  unset(_proxsuite_nlp_python_bindings_not_found)
-
-  execute_process(
-    COMMAND ${PYTHON_EXECUTABLE} -c "import pinocchio"
-    RESULT_VARIABLE _pinocchio_python_bindings_not_found
-    OUTPUT_QUIET ERROR_QUIET
-  )
-  if(_pinocchio_python_bindings_not_found EQUAL 0)
-    message(STATUS "Found pinocchio Python bindings.")
-    set(BUILD_WITH_PINOCCHIO_PYTHON_BINDINGS True)
-  else()
-    message(STATUS "pinocchio Python bindings NOT found.")
-  endif()
-  unset(_pinocchio_python_bindings_not_found)
-
-endif(BUILD_PYTHON_INTERFACE)
+endif()
 
 add_project_dependency(proxsuite-nlp 0.6.1 REQUIRED PKG_CONFIG_REQUIRES "proxsuite-nlp >= 0.6.1")
 
-- 
2.34.1

